documentation/staging/content/_index.md

```diff
@@ -63,3 +63,4 @@ See [Get help]({{< relref "userguide/introduction/get-help.md" >}}).
 * [WebLogic Monitoring Exporter](https://github.com/oracle/weblogic-monitoring-exporter)
 * [WebLogic Logging Exporter](https://github.com/oracle/weblogic-logging-exporter)
 * [WebLogic Remote Console](https://github.com/oracle/weblogic-remote-console)
+* [WebLogic Kubernetes Toolkit UI](https://oracle.github.io/weblogic-toolkit-ui/)
```

Comment:
I'm assuming that these are not just changes that you want to go in for the next release, but instead changes that you want to go in *right now* and be preserved for the next release. Therefore, these changes need to go in to the matching files in both documentation/staging and documentation/3.3.

Reply:
ok, will do.

Reply:
Does the date fix (11/9) need to go under documentation/3.3 also?

Reply:
Yep, did this also.